streampark-console/streampark-console-webapp/src/views/user/SignIn.vue

```diff
@@ -86,6 +86,21 @@
             </a-button>
           </a-form-item>
         </a-form>
+        <a-form-item
+          style="margin-top:40px">
+          <a-button
+            style="padding: 0 90px"
+            size="large"
+            type="primary"
+            html-type="button"
+            class="signin-button"
+            href="http://localhost:7001/login/oauth/authorize?client_id=b6f7a3bdee5d3f424ee2&response_type=code&redirect_uri=http://localhost:10003&scope=read&state=app-test"
```

Comment:
Maybe it would be better if this parameter can be configured

Reply:
ok

Reply:
You can configue it in main.js now